scripts/verify-jenkins-on-openshift.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+# This script deploys the jenkins pod with provided image (from CPaas) on an 
+# OpenShift cluster & executes verify_jenkins.sh within the newly deployed pod.
+set -o pipefail
+
+# Takes input & stores in the respective variables
+# 
+get_input(){
+    read -p "OpenShift's API URL: " API_SERVER
+    read -p "OpenShift's User Name (kubeadmin): " USER_NAME
+    read -p "OpenShift's User Password: " USER_PASSWORD
+    read -p "Jenkins Image built by CPaas: " JENKINS_IMAGE  
+}
+
+vars(){
+
+    get_input
+    IFS=":" read -r IMG TAG <<< ${JENKINS_IMAGE}
+    QUAY_IMAGE="quay.io/pipeline-integrations/openshift-ose-jenkins:${TAG}"
+
+    # if username is not passed, pick `kubeadmin` as default user
+    if [ -z ${USER_NAME} ]
+    then
+        USER_NAME="kubeadmin"
+    fi
+}
+
+# get_image_ready() 
+# pulls the brew image locally, tags & pushes it to quay.
+# 
+get_image_ready(){
+    echo -e "\n-------------------------------------------------------\n>INFO || Pulling the Jenkins Image\n"
+    IMAGE_ID=$(podman pull ${JENKINS_IMAGE})
+    if [ -z $IMAGE_ID ]
+    then
+        echo -e "\n>ERR =======> Image Pull Failed\n"
+        exit 1
+    else
+        podman tag ${JENKINS_IMAGE} ${QUAY_IMAGE}
+        echo -e "\n-------------------------------------------------------\n>INFO || Pushing the image to quay.io/pipeline-integrations\n"
+        podman push ${QUAY_IMAGE}
+        echo -e "\n-------------------------------------------------------\n>INFO || Image push successful\n"
+    fi
+}
+
+# deploy_on_openshift()
+# Deploys the pod with the provided CPaas image, 
+# 
+deploy_on_openshift(){
+
+    # ensure oc exists
+    oc version > /dev/null
+    if [ $? != 0 ]
+    then
+        echo "\n-------------------------------------------------------> Please install `oc` client to continue. https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/"
+        exit 1
+    fi
+
+    # if oc exists, continue
+    echo -e "\n-------------------------------------------------------\n>INFO || Logging in as " ${USER_NAME} " to " ${API_SERVER} "\n"
```

Comment:
printf is usually preferred instead of echo, especially when you are including variables in the output.